### DISCUSSION TITLE
how to disable a certain check for all Dockerfiles

### DISCUSSION THREAD

**Discussion by uwebartels** [Checkmarx/kics#6776] [General] [answered]
Hello,

I'd like to disable a single check across all files. Based on the docs I cannot see how to do it.
I am introducing kics via metalinter to a repo with many Dockerfiles. So for the start I want to disable the cheeck HEALTHCHECK as I get too many errors to fix this in some weeks.

The Error is:
Description: Ensure that HEALTHCHECK is being used. The HEALTHCHECK instruction tells Docker how to test a container to check that it is still working

Thanks,
Uwe

**Comment by gabriel-cx** [comment, marked as answer]
Hi @uwebartels ,

Thanks for your inputs!
I assume you are referring to this [query](https://github.com/Checkmarx/kics/blob/947ca37d4ddae175b36820e54115e2e5e4058ccc/assets/queries/dockerfile/healthcheck_instruction_missing/metadata.json#L3).

In order to disable that specific query you can use the KICS flag `--exclude-queries`, kindly check it [here](https://docs.kics.io/latest/commands/#scan_command_options).
